tests/sqllogictests/suites/crdb/and_or

```diff
@@ -7,18 +7,18 @@ CREATE TABLE t (k INT null, a INT null, b INT null)
 statement ok
 INSERT INTO t VALUES (1, NULL, NULL), (2, NULL, 1), (3, 1, NULL), (4, 2, 0), (5, 3, 3)
 
-statement error 1001
+statement error 1001.*divided by zero while evaluating function `divide\(3, 0\)`
 SELECT a <> 2 AND 3 / b = 1 FROM t ORDER BY k
 
 query I
 SELECT a FROM t WHERE a <> 2 AND 3 / b = 1 ORDER BY k
 ----
 3
 
-statement error 1001
+statement error 1001.*divided by zero while evaluating function `divide\(3, 0\)`
```

Comment:
can we omit `.*` to be spaces?

And we only use string contains to match the error text.

Reply:
the string after `error` is regex.
currently error code is not used, but maybe we need to standardize ant test it later.

if we need string match only,   it is better if sqllogictest-rs support `error_substring` directly so we don`t need to escape.